.github/workflows/test.yml

```diff
@@ -35,6 +35,7 @@ jobs:
     - name: Run tests
       run: |
         hatch -v run test
+        hatch -v run test_readme
```

Comment:
Meh, it is that useful to test the README ?

Reply:
One less thing that can go wrong!